libASL/dis.ml

```diff
@@ -951,6 +951,18 @@ and dis_lexpr_chain (loc: l) (x: lexpr) (ref: access_chain list) (r: sym): unit
                 (* this loses propagation of pure expressions when they are assigned
                    into structures, but this is unavoidable since the structure value types
                    cannot store expressions. *)
+                
+                (* mark all pstate.el or pstate.sp writes as unsupported and die when we see them.
+                   this basically "fixes" us in EL0 and eliminates a bunch of branches.
+                   fun fact - the only instructions i'm aware of that can actually do this don't
+                   work anyway *)
+                let () = (match var, ref with
+                | Var(0, Ident("PSTATE")), ([Field(Ident("EL"))] | [Field(Ident("SP"))]) ->
```

Comment:
I'm actually not sure if this is guaranteed to be OK, because I'm not sure if the Field constructor is enough to give us something that'll match a "complete" access chain.